src/prodbg/main/src/windows.rs

```diff
@@ -150,6 +175,7 @@ impl Windows {
                   sessions: &mut Sessions,
                   view_plugins: &mut ViewPlugins,
                   backend_plugins: &mut BackendPlugins) {
+        sleep(Duration::from_millis(100));
```

Comment:
But sleeping for 100 ms lowers the framerate to 10 fps. Sleeping for 1 ms is likely more reasonable.